service/tablet_allocator.cc

```diff
@@ -383,7 +383,7 @@ class load_balancer {
 
     struct skipped_candidate {
         tablet_replica replica;
-        global_tablet_id tablet;
+        tablet_candidate tablet;
```

Comment:
I remember I was willing to rename things to 'tablet_candidate candidate', but to reduce changes and make review easier I decided to keep it. If we rename it to tablet_candidate_set, then we name it either tablets or simply candidate. what about migration_candidate? even thought it might span two tablet replicas, we can think of it as a abstraction to some logical unit to be migrated. then existing migration_candidate needs to be renamed to migration_candidate_descriptor or simply migration_descriptor. what do you think?